R/stat-qq-line.R

```diff
@@ -71,9 +71,7 @@ StatQqLine <- ggproto("StatQqLine", Stat,
 
     theoretical <- inject(distribution(p = quantiles, !!!dparams))
 
-    if (length(line.p) != 2) {
-      cli::cli_abort("Cannot fit line quantiles {line.p}. {.arg line.p} must have length 2.")
-    }
+    check_length(line.p, 2L)
```

Comment:
It seems this message is now less descriptive

Reply:
Yeah somewhat. Do you prefer to revert, ignore this, or wrap in `try_fetch()` block and appending the offending quantile line?